### PR TITLE
Darlp networks 1201256994704235

### PR DESCRIPTION
**Pull request by darlp5589**
Hi @chriscodev @triveni-codev 

Kindly check this PR. 

Here's the ASANA ticket details: https://app.asana.com/0/1201943683850147/1201256994704235

So basically, this should enable the feature to allow only 1 transport to 1 network association.

Take note this is a branch out from PR432.

### CODE REVIEW

File: src/app/pages/dashboard/network-config/networks/networks.component.ts [hunk 1]
```diff
@@ -243,19 +254,33 @@ export class NetworksComponent implements OnInit {
 							const ipsettings = this.tableNetworksData;
 							let dataRow = ipsettings.find((obj) => obj.id === tableID);							
 							let dataValues = {};
+							if(dataRow === undefined) {
```
Comment (chriscodev) [inline src/app/pages/dashboard/network-config/networks/networks.component.ts:257]: if unused remove line code

File: src/app/pages/dashboard/network-config/networks/networks.component.ts [hunk 2]
```diff
@@ -339,22 +364,35 @@ export class NetworksComponent implements OnInit {
...
+					descriptorElements: descriptorElementsArray,
 					channels: this.channelList,
 					networkType: this.networksDataObject.selectedtype,
 				};
 
 				if(GlobalVariables.networksCommitType == 'add') {
-					data['id'] = this.tableNetworksData.length + 1;
+					// data['id'] = this.tableNetworksData.length + 1;
+					// data['clientId'] = this.tableNetworksData.length + 1;
+					data['id'] = new Date().getTime();
```
Comment (chriscodev) [inline src/app/pages/dashboard/network-config/networks/networks.component.ts:379]: this should be stored in 1 variable = new Date().getTime(); to optimize calling of function call,a nd to  be resusable

**Review by chriscodev — COMMENTED** [review]
new PR updates - general feedback

File: src/app/pages/dashboard/network-config/networks/networks.component.ts (hunk 2 above)
Comment (darlp5589) [reply src/app/pages/dashboard/network-config/networks/networks.component.ts:379]: Oh yeah! Definitely make sense! Thanks! Fixed!

**Comment by chriscodev** [comment]
Commit: https://github.com/triveni-codev/GBWebApp-dev/pull/438/commits/a43ab28e9bce4ed0b3f12c4498b966489149411f

LGTM

**Comment by triveni-codev** [comment]
PR resynced (where approved changes from previous PR is removed from diff) to reduce the number of files changed from 14 to  4

Then:
image

Now:
image

**Comment by triveni-codev** [comment]
Merging this PR. This has been further discussed with Darl.

~~~~~~~~~~~~~
Networks Section: Establishment of "one Transport is to one Network" relationship
https://app.asana.com/0/1201448109049348/1201256994704235/f

**Pull request merged**